.babelrc.js

```diff
@@ -0,0 +1,81 @@
+const env = process.env.NODE_ENV;
+const buildIgnore = [
+  '*.jest.js',
+  '*.e2e.js',
+  '*.ssr.js',
+  '*.example.js',
+  'source/demo',
+  'source/jest-*.js',
+  'source/TestUtils.js',
+];
+
+if (env === 'commonjs') {
+  module.exports = {
+    ignore: buildIgnore,
+    plugins: [
+      'transform-runtime',
+      ['flow-react-proptypes', {deadCode: true}],
+      ['transform-react-remove-prop-types', {mode: 'wrap'}],
+    ],
+    presets: ['env', 'react', 'flow', 'stage-1'],
+  };
+}
+
+if (env === 'es') {
+  module.exports = {
+    ignore: buildIgnore,
+    plugins: [
+      'transform-runtime',
+      ['flow-react-proptypes', {deadCode: true}],
+      ['transform-react-remove-prop-types', {mode: 'wrap'}],
+    ],
+    presets: [['env', {modules: false}], 'react', 'flow', 'stage-1'],
+  };
+}
+
+if (env === 'rollup') {
+  module.exports = {
+    comments: false,
+    plugins: ['external-helpers'],
```

Comment:
technically this could be passed to rollup-plugin-babel and it would just merge it with what this `.babelrc.js` returns, that way you wouldn't have introduce separate env var just for rollup, but u'd somewhat loose 'centralization' of all babel configuration 

i wouldnt concern much about the latter as its kinda temporary, because with babel@7 and its corresponding version of the babel plugin you won't have to deal with external-helpers at all, this optimization will be applies automatically by the plugin

Reply:
Yeah, I'm gonna open PR with babel7 later